Formula/procyon-decompiler.rb

```diff
@@ -1,34 +0,0 @@
-class ProcyonDecompiler < Formula
-  desc "Modern decompiler for Java 5 and beyond"
-  homepage "https://bitbucket.org/mstrobel/procyon/wiki/Java%20Decompiler"
-  url "https://bitbucket.org/mstrobel/procyon/downloads/procyon-decompiler-0.5.36.jar"
-  sha256 "74f9f1537113207521a075fafe64bd8265c47a9c73574bbf9fa8854bbf7126bc"
-  revision 1
-
-  disable! date: "2020-08-22", because: :repo_removed
```

Comment:
Perhaps worth noting that this isn't quite true. The repo redirects here: https://github.com/mstrobel/procyon.

The release JAR however was not migrated - only a 0.6 prerelease is available.